pkg/controller/argocd/rolebinding.go

```diff
@@ -83,68 +83,79 @@ func (r *ReconcileArgoCD) reconcileRoleBindings(cr *argoprojv1a1.ArgoCD) error {
 }
 
 func (r *ReconcileArgoCD) reconcileRoleBinding(name string, rules []v1.PolicyRule, cr *argoprojv1a1.ArgoCD) error {
```

Comment:
Same here as with `reconcileRole()`. Please document it. Help others understanding it.

Reply:
added comments!